feature-libs/organization/administration/organization.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { OrganizationComponentsModule } from '@spartacus/organization/administration/components';
+import { OrganizationCoreModule } from '@spartacus/organization/administration/core';
+import { OrganizationOccModule } from '@spartacus/organization/administration/occ';
+
+@NgModule({
+  imports: [
+    OrganizationCoreModule.forRoot(),
+    OrganizationOccModule,
+    OrganizationComponentsModule,
+  ],
+})
+export class OrganizationModule {}
```

Comment:
Should this folder be still called `OrganizationModule`? Maybe `AdministrationModule` or `OrganizationAdministrationModule`?

(same for child modules ...Core, Occ,Components)

Reply:
We agreed offline to do it in the next iteration.